plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -237,8 +242,23 @@ func init() {
 	})
 }
 
-func createRealIngestor(client localClient, database string, tableName string) (localIngestor, error) {
-	ingestor, err := ingest.New(client.(*kusto.Client), database, tableName, ingest.WithStaticBuffer(bufferSize, maxBuffers))
+// For each table create the ingestor
+func createIngestorByTable(client *kusto.Client, database string, tableName string, ingestionType string) (ingest.Ingestor, error) {
+	var ingestor ingest.Ingestor
+	var err error
+	if strings.ToLower(ingestionType) == managedIngestion {
```

Comment:
Maybe also add a constant for queued, so the users could specify either one, and anything else would be an error.